### PR TITLE
Added pyTorch to test run

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
Build internally 32410.

File: docs/examples/pytorch/pytorch-resnet50.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ Requirements
 .. role:: bash(code)
    :language: bash
 
-- `APEx <https://www.github.com/nvidia/apex>`_
+- `APEx <https://www.github.com/nvidia/apex>`_ - optional, required for fp16 mode or distirbuted (mutli GPU) operation
```
Comment (ptrendx) [inline docs/examples/pytorch/pytorch-resnet50.rst]: "distributed"

File: docs/examples/pytorch/pytorch-resnet50.rst [hunk 2]
```diff
@@ -52,7 +52,7 @@ Usage
    PyTorch ImageNet Training
 
    positional arguments:
-   DIR                         path to dataset
+   DIR                         path/s to dataset (one path assumes train and val inside, two train and val need to be pointed directly)
```
Comment (ptrendx) [inline docs/examples/pytorch/pytorch-resnet50.rst]: This is unclear.

File: docs/examples/pytorch/pytorch-resnet50.rst (hunk 1 above)
Comment (cliffwoolley) [reply docs/examples/pytorch/pytorch-resnet50.rst]: Sorry, my review from a few hours ago had been left in a pending state.  See below.

File: docs/examples/pytorch/pytorch-resnet50.rst (hunk 2 above)
Comment (cliffwoolley) [reply docs/examples/pytorch/pytorch-resnet50.rst]: Suggestion below.

File: docs/examples/pytorch/main.py [hunk 3]
```diff
@@ -38,8 +32,9 @@
                      and callable(models.__dict__[name]))
 
 parser = argparse.ArgumentParser(description='PyTorch ImageNet Training')
-parser.add_argument('data', metavar='DIR',
-                    help='path to dataset')
+parser.add_argument('data', metavar='DIR', nargs='*',
+                    help='path/s to dataset (one path assumes train and val inside,\n' +
+                         'two tain and val are pointed directly)')
```
[the remark this answers is not in the document]
Comment (JanuszL) [reply docs/examples/pytorch/main.py]: Done

File: docs/examples/pytorch/pytorch-resnet50.rst (hunk 2 above)
Comment (JanuszL) [reply docs/examples/pytorch/pytorch-resnet50.rst]: Done

File: docs/examples/pytorch/pytorch-resnet50.rst (hunk 1 above)
Comment (JanuszL) [reply docs/examples/pytorch/pytorch-resnet50.rst]: Done

File: docs/examples/pytorch/pytorch-resnet50.rst [hunk 4]
```diff
@@ -20,7 +20,7 @@ Requirements
 .. role:: bash(code)
    :language: bash
 
-- `APEx <https://www.github.com/nvidia/apex>`_
+- `APEx <https://www.github.com/nvidia/apex>`_ - optional, required for fp16 mode or distributed (mutli GPU) operation
```
Comment (cliffwoolley) [inline docs/examples/pytorch/pytorch-resnet50.rst]: multi

(`multi-GPU` would be best)

**Pull request merged**